omero/util_scripts/Dataset_To_Plate.py

```diff
@@ -39,9 +39,9 @@
 from omero.rtypes import rint, rlong, rstring, robject, unwrap
 
 
-def add_image_to_plate(conn, image, plate_id, column, row, remove_from=None):
+def add_images_to_plate(conn, images, plate_id, column, row, remove_from=None):
```

Comment:
I assume as this is targetted for OMERO 5.5, changing this API is fine although if we wanted to maintain compatibility for anyone who consumed this API, it should be straightforward to proxy the old API/signature as:
 

```
def add_image_to_plate(conn, image, plate_id, column, row, remove_from=None):
    return add_images_to_plate(conn, [image], plate_id, column, row, remove_from=remove_from)
```

Reply:
I don't think that internal script methods will be used externally. Better to keep this simple since the main function of these scripts is as examples and an unused ```add_image_to_plate()``` is confusing.